mappings/net/minecraft/world/gen/HeightContext.mapping

```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/class_5868 net/minecraft/world/gen/HeightContext
+	FIELD field_34030 minY I
+	FIELD field_34031 maxY I
```

Comment:
Isn't this height?

Reply:
Lemme see, if this is the case getters need renaming

Reply:
Yeah, seems like the getter needs to be fixed.

Reply:
I recall we mostly use `height` as an alternative to `deltaY`; it is a relative than an abstract value

Reply:
Okay, confirmed previous mapping is wrong and this is a relative value (YOffset uses it in BelowTop as `method_30458() + getMinY() - 1 - offset`

Reply:
Ah thank you guys for correcting :heart: